samples/benchmarks/coremark/Kconfig.sysbuild

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "share/sysbuild/Kconfig"
```

Comment:
Can we move `rsource` before the `source`? (to allow overriding sourced symbols and for consistency with sample's Kconfig)